src/lib.rs

```diff
@@ -4,8 +4,8 @@
 #![recursion_limit = "256"]
 #![allow(clippy::match_like_matches_macro)]
 
-#[macro_use]
-extern crate tracing;
+// #[macro_use]
+// extern crate tracing;
```

Comment:
why jsut commented out?

Reply:
@calebcartwright no idea. It was added in https://github.com/rust-lang/rust/pull/129774. I can remove this.